### PR TITLE
fc-app 157: Cromwell's /logs endpoint does not always have a 'calls' field

### PR DESCRIPTION
**Pull request by jmthibault79**
- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check that the **Product Owner** has signed off on any user-facing changes
- [ ] **Submitter**: Make sure Swagger is updated if API changes
  - [ ] **...and Orchestration's Swagger too!**
- [ ] **Submitter**: If updating admin endpoints, also update [firecloud-admin-cli](https://github.com/broadinstitute/firecloud-admin-cli)
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Database checks:
  * If PR includes new or changed db queries, include the explain plans in the description
  * Make sure liquibase is updated if appropriate
  * If doing a migration, take a backup of the
  [dev](https://console.cloud.google.com/sql/instances/terraform-qfarbdq3lrexxck5htofjs5z6m/backups?project=broad-dsde-dev&organizationId=548622027621)
  and
  [alpha](https://console.cloud.google.com/sql/instances/terraform-r4caezzc35c4tb7pgdhwkmme4y/backups?project=broad-dsde-alpha&organizationId=548622027621)
  DBs in Google Cloud Console
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] Tell your tech lead (TL) that the PR exists if they want to look at it
- [ ] Anoint a lead reviewer (LR). **Assign PR to LR**
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Assign to submitter** to finalize
- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Squash commits and merge to develop
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Verify swagger UI on dev environment still works after deployment
- [ ] **Submitter**: Inform other teams of any API changes via Slack and/or email
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: automation/project/Dependencies.scala [hunk 1]
```diff
@@ -7,7 +7,7 @@ object Dependencies {
   val akkaV = "2.5.7"
   val akkaHttpV = "10.0.10"
 
-  val serviceTestV = "0.10-61981d5"
+  val serviceTestV = "0.12-9478f7e-SNAP"
```
Comment (jmthibault79) [inline automation/project/Dependencies.scala]: Depends on https://github.com/broadinstitute/workbench-libs/pull/158

File: automation/src/test/scala/org/broadinstitute/dsde/test/api/RawlsApiSpec.scala [hunk 2]
```diff
@@ -130,7 +130,7 @@ class RawlsApiSpec extends TestKit(ActorSystem("MySpec")) with FreeSpecLike with
       }
     }
 
-    "should retrieve sub-workflow metadata from Cromwell" in {
+    "should retrieve sub-workflow metadata and outputs from Cromwell" in {
```
Comment (jmthibault79) [inline automation/src/test/scala/org/broadinstitute/dsde/test/api/RawlsApiSpec.scala:133]: Could add a new test for this instead.

File: core/src/test/scala/org/broadinstitute/dsde/rawls/mock/RemoteServicesMockServer.scala [hunk 3]
```diff
@@ -654,9 +654,7 @@ class RemoteServicesMockServer(port:Int) extends RawlsTestUtils {
           .withBody(
             """
               |{
-              |  "id": "8afafe21-2b70-4180-a565-748cb573e10c",
-              |  "calls": {
```
Comment (jmthibault79) [inline core/src/test/scala/org/broadinstitute/dsde/rawls/mock/RemoteServicesMockServer.scala:658]: When there are no calls, Cromwell omits this field entirely.

**Review by davidangb — APPROVED** [review]
:+1: assuming tests pass.

In the test spec and mock, you could use `Some(Map ...` instead of `Option(Map ...` but that's not a big deal.

**Comment by jmthibault79** [comment]
Travis failure is `GoogleGroupSyncMonitorSpec` which always fails so don't worry about that

**Review by davidangb — APPROVED** [review]
bah. still :+1:, assuming tests pass.

**Comment by jmthibault79** [comment]
Rawls auto-test passes.  Only auto-test failure is `MethodConfigSpec` "stays current with selected root entity type"

**Comment by jmthibault79** [comment]
Rerunning the one failed auto-test [passed](https://fc-jenkins.dsp-techops.broadinstitute.org/job/swatomation-pipeline/8604/).

**Pull request merged**